src/EditorComponent.jsx

```diff
@@ -160,6 +161,14 @@ class StoredTimestamp {
     }
 }
 
+class SaveCurrentNoteOptions {
```

Comment:
Why do we need this? Can't we use plain objects?

Reply:
I mainly use these classes since they make the object schema visible in code and I can search the file for usages. I realize that that really isn't much especially file local objects like these. OK, going to increase my usage of plain objects for these cases.